backend/apps/vacancies/api/v1/serializers.py

```diff
@@ -0,0 +1,35 @@
+from apps.vacancies.models import Currency, Vacancy
+from django.utils.translation import gettext_lazy as _
+from rest_framework import serializers
+from taggit.serializers import TaggitSerializer, TagListSerializerField
+
+
+class VacancySerializer(serializers.ModelSerializer, TaggitSerializer):
+    """Serializer for CRUD Cacancy endpoint."""
+
+    keywords = TagListSerializerField()
+    end_date = serializers.DateField()
```

Comment:
I guess you don't need to explicitly mark those fields, as you already put "__all__" in Meta class. Now you are just redefining the fields. You should only do it for custom fields or if you need to change default settings.